### PR TITLE
[UTOPIA-1471] - Set up a basic Playwright environment for testing

### PR DESCRIPTION
**Pull request by NoorChasib**
## 🎯 Summary

  
[UTOPIA-1471](https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-1471)

- created new Playwright structure
- initialized new dependencies
- configured playwright config for testing environment
- created new scripts to run tests
- created example test to check if everything is running smoothly

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: src/frontend/package.json [hunk 1]
```diff
@@ -25,6 +37,7 @@
     "@fortawesome/react-fontawesome": "0.2.0",
     "@uiw/react-md-editor": "3.24.1",
     "breakpoint-sass": "3.0.0",
+    "dotenv": "16.3.1",
```
Comment (BradyMitch) [inline src/frontend/package.json:40]: If we update to node 20.6, dotenv is no longer required to access process.env

**Comment by BradyMitch** [comment]
Can we separate the e2e tests into `src/e2e`? There's no reason to include them in frontend and it just adds to the amount of files that need to be copied when building the frontend.

File: src/frontend/package.json (hunk 1 above)
Comment (NoorChasib) [reply src/frontend/package.json:40]: are we planning on moving soon?

**Comment by NoorChasib** [comment]
> Can we separate the e2e tests into `src/e2e`? There's no reason to include them in frontend and it just adds to the amount of files that need to be copied when building the frontend.

yeah we can make that change, ill try and see if it acts weird to have it live in the parent folder but ill push a change soon if not

**Comment by BradyMitch** [comment]
Note: We have to run `npx playwright install` before running tests for the first time.

File: src/frontend/package.json (hunk 1 above)
Comment (BradyMitch) [reply src/frontend/package.json:40]: We're already on 20.5 I think so not hard to update.

File: src/frontend/package.json [hunk 2]
```diff
@@ -36,6 +49,8 @@
     "@babel/core": "7.23.2",
     "@babel/preset-env": "7.23.2",
     "@babel/register": "7.22.15",
+    "@playwright/test": "1.39.0",
```
Comment (BradyMitch) [inline src/frontend/package.json:52]: Make sure to remove the changes to frontend package.json and npm install again

**Review by BradyMitch — APPROVED** [review]
We should probably come up with an eslint and prettier config for this

**Pull request merged**